base/setup/usetup/lang/en-US.h

```diff
@@ -109,7 +109,7 @@ static MUI_ENTRY enUSWelcomePageEntries[] =
     {
         6,
         11,
-        "This part of the setup copies the ReactOS Operating System to your",
+        "This part of the setup copies the ReactOS to your",
```

Comment:
What about the other languages in usetup (and "welcome")? Do they have (in their translated way) have "ReactOS operating system"?